### PR TITLE
Compliance Operator v0.1.49 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Will need cherry-pick to 4.6+ after the errata goes live.
Preview: https://deploy-preview-44510--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-0-1-49-new-features-and-enhancements

### CODE REVIEW

**Review by EricPonvelle — COMMENTED** [review]
I had a couple of wording suggestions

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +13,39 @@ These release notes track the development of the Compliance Operator in the {pro
...
+
+* Previously, the `ocp4-cluster-version-operator-verify-integrity` always checked the first entry in the Cluter Version Operator (CVO) history. As a result, the upgrade would fail in situations where subsequent versions of {product-name} would be verified. Now, the compliance check result for `ocp4-cluster-version-operator-verify-integrity` is able to detect verified versions and is accurate with the CVO history. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053602[*BZ#2053602*])
+
+* Previously, the `ocp4-api-server-no-adm-ctrl-plugins-disabled` rule did not check for a list of empty admission controller plugins. As a result, the rule would always fail, even if all admission plugins were enabled. Now, more robust checking of the `ocp4-api-server-no-adm-ctrl-plugins-disabled` rule will accurately pass with all admission controller plugins enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2058631[*BZ#2058631*])
```
Comment (ahardin-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:45]: s/plugins/plug-ins

**Pull request merged**